Assets/Hamster/Scripts/MainGame.cs

```diff
@@ -192,30 +192,23 @@ public class MainGame : MonoBehaviour {
       // Feature Flags
       defaults.Add(StringConstants.RemoteConfigGameplayRecordingEnabled, false);
 
-      Firebase.RemoteConfig.FirebaseRemoteConfig.SetDefaults(defaults);
-      return Firebase.RemoteConfig.FirebaseRemoteConfig.FetchAsync(System.TimeSpan.Zero);
+      Firebase.RemoteConfig.FirebaseRemoteConfigDeprecated.SetDefaults(defaults);
```

Comment:
I understand the purpose of this cl is to make MH build but...
It really should not be too difficult to use the new API.

Ex. This line should be easily replaced with:
```
return Firebase.RemoteConfig.FirebaseRemoteConfig.DefaultInstance.SetDefaultAsync(defaults).ContinueWithOnMainThread( task => {
  // check if it failed...
  return Firebase.RemoteConfig.FirebaseRemoteConfig.DefaultInstance.FetchAsync();
}).UnWrap();
```